### PR TITLE
SK-812 Add options (tokens) support for Get methods

### PR DESCRIPTION
**Pull request by skyflow-lipsa**

### CODE REVIEW

File: skyflow/vault/_config.py [hunk 1]
```diff
@@ -24,37 +24,55 @@ def __init__(self, vaultID: str=None, vaultURL: str=None, tokenProvider: Functio
...
+    def __init__(self, tokens: bool = True):
         self.tokens = tokens
 
+
+class GetOptions:
+    def __init__(self, tokens: bool = False):
+        self.tokens = tokens
+
+
+# class GetOptions:
```
Comment (skyflow-vivek) [inline skyflow/vault/_config.py:50]: remove commented code.

File: tests/vault/test_get.py [hunk 2]
```diff
@@ -121,68 +121,89 @@ def testGetByIdInvalidTableType(self):
...
             self.assertEqual(e.code, SkyflowErrorCodes.INVALID_INPUT.value)
             self.assertEqual(
-                e.message, SkyflowErrorMessages.INVALID_REDACTION_TYPE.value % (str))
+                e.message, SkyflowErrorMessages.INVALID_COLUMN_NAME.value % (list))
 
-    def testGetByIdNoColumnName(self):
+    def testGetByIdNoColumnValues(self):
         invalidData = {"records": [
-            {"table": "pii_fields", "redaction": RedactionType.PLAIN_TEXT}]}
+            # {"table": "pii_fields", "redaction": RedactionType.PLAIN_TEXT, "columnName": "first_name"}]}
```
Comment (skyflow-vivek) [inline tests/vault/test_get.py:149]: remove commented line

File: samples/get_with_options.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        '<YOUR_VAULT_ID>', '<YOUR_VAULT_URL>', token_provider)
+    client = Client(config)
+    options = GetOptions(False)
+    data = {"records": [
+        {
+            "ids": ["<SKYFLOW_ID1>", "<SKYFLOW_ID2>", "<SKYFLOW_ID3>"],
+            "table": "<TABLE_NAME>",
+            "redaction": RedactionType.PLAIN_TEXT
+        },
+        # To get records using unique column name and values.
```
Comment (skyflow-vivek) [inline samples/get_with_options.py:29]: remove commented code.

**Review by skyflow-vivek — APPROVED** [review]

File: skyflow/vault/_config.py (hunk 1 above)
Comment (skyflow-lipsa) [reply skyflow/vault/_config.py]: ok

**Review by skyflow-vivek — APPROVED** [review]

**Review by skyflow-bharti — APPROVED** [review]

**Review by yaswanth-pula-skyflow — APPROVED** [review]

**Pull request merged**